### ISSUE TITLE
task textarea is not resizable

### ISSUE THREAD

**Issue by asurkov** [Igalia/phpreport#468]
If you need to enter a lot of details into task textarea, it's tempting to resize textarea to avoid scrolling, which is a default behavior of textarea element btw.

However task textarea has resize:none CSS style applied which prevents it from being resizable.

**Comment by jaragunde** [comment]
Thanks for the hint, it is indeed a simple change. `resize: vertical` seems appropriate, to keep the general aspect and prevent the resizing corner being hidden by mistake.

**Comment by jaragunde** [comment]
```
commit c8b4cf70062c108ca936b97f70ad050d1ae4a963
Author: Jacobo Aragunde Pérez <jaragunde@igalia.com>
Date:   Fri Feb 28 12:33:18 2020 +0100

    [#468] Allow to resize the task textarea.
    
    Allow vertical resize only to keep the general aspect and to prevent
    hiding the resizing corner by mistake.

```

**Comment by jaragunde** [comment]
Closed #163 as duplicate.
